src/components/WriteEditModal/WriteEditModal.tsx

```diff
@@ -335,7 +397,7 @@ export default function WriteEditModal(props: WriteEditModalProps) {
             disabled={isCustom ? false : rating === 0}
             backgroundColor={isValid && rating > 0 ? 'background-primary' : 'background-gray-40'}
           >
-            등록
+            {reviewType === 'productReview' ? '등록' : '수정'}
```

Comment:
이거 customReview 일 때도 등록으로 해주면 좋을 것 같아요!!

```suggestion
            {reviewType === 'productReview' || 'customReiview'  ? '등록' : '수정'}
```